src/Sylius/Behat/Context/Setup/ProductReviewContext.php

```diff
@@ -45,14 +45,15 @@ public function productHasAReview(ProductInterface $product, CustomerInterface $
 
     /**
      * @Given /^(this product) has(?:| also) a review titled "([^"]+)" and rated (\d+) added by (customer "[^"]+")(?:|, created (\d+) days ago)$/
+     * @Given /^(this product) has(?:| also) an accepted review titled "([^"]+)" and rated (\d+) added by (customer "[^"]+")(?:|, created (\d+) days ago)$/
      */
-    public function thisProductHasAReviewTitledAndRatedAddedByCustomer(
+    public function thisProductHasAnAcceptedReviewTitledAndRatedAddedByCustomer(
         ProductInterface $product,
-        $title,
-        $rating,
+        string $title,
+        int $rating,
         CustomerInterface $customer,
-        $daysSinceCreation = null,
-    ) {
+        ?int $daysSinceCreation = null,
+    ): void {
         $review = $this->createProductReview($product, $title, $rating, $title, $customer);
         if (null !== $daysSinceCreation) {
```

Comment:
```suggestion

        if (null !== $daysSinceCreation) {
```

Reply:
Fixed in https://github.com/Sylius/Sylius/pull/16325